cram_3d_world/cram_bullet_reasoning/src/objects.lisp

```diff
@@ -155,7 +155,7 @@
   (dolist (body (rigid-bodies object))
     (draw context body)))
 
-(defun make-rigid-body-name (obj-name body-name)
+(defun make-rigid-body-name (obj-name body-name &optional (package (sb-int:sane-package)))
```

Comment:
Could you change ```(sb-int:sane-package)``` to ```*package*```?
It does the same but ```*package*``` is SBCL-independent, works on any Lisp.

Reply:
Removed the parameter again. Instead added :keyword as the package parameter to intern.